### PR TITLE
update error catch and Get return

### PR DESCRIPTION
**Pull request by mgreenegit**
In version 1, if the copy failed during Set the error was not properly caught and displayed.  Also updating Get so that properties are displayed based on input.  In most cases, I would not use input as what should be returned by Get, but after working with this resource I realize that is the only way to report on which parameters are being passed to the command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,8 +74,12 @@ Versions
 --------
 
 ### 1.0.0.0
-* Initial public release of xRobocopy module with following resources:
-	* xRobocopy
+ - Initial public release of xRobocopy module with following resources:
+    - xRobocopy
+ 
+### 1.1.0.0
```
Comment (KarolKaczmarek) [inline README.md:80]: this should be added as "Unreleased" section. We will bump the version number when we release to PS Gallery

File: DSCResources/MSFT_xRobocopy/MSFT_xRobocopy.psm1 [hunk 2]
```diff
@@ -13,15 +13,53 @@ function Get-TargetResource
...
+        Retry = $Retry
+        Wait = $Wait
+        SubdirectoriesIncludingEmpty = $SubdirectoriesIncludingEmpty
+        Restartable = $Restartable
+        Multithreaded = $MultiThreaded
+        ExcludeFiles = $ExcludeFiles
+        ExcludeDirs = $ExcludeDirs
+        LogOutput = $LogOutput
+        AppendLog = $AppendLog
+        AdditionalArgs = $AdditionalArgs
```
Comment (KarolKaczmarek) [inline DSCResources/MSFT_xRobocopy/MSFT_xRobocopy.psm1:62]: Get-TargetResource is supposed to return the actual state of the machine.
In current implementation it returns whatever user wants it to be (whatever was specified in the configuration), but without saying what is the real state. 

I'd suggest following pattern used by other resources, adding readonly property "Ensure" which would contain information (Boolean) if file described in Destination path exists on the given node or not.

see issue #6

**Comment by mgreenegit** [comment]
That makes sense.  Only for Ensure, correct?  Otherwise the details of the configuration are not stored anywhere and the original input is actually very important to know.

**Comment by KarolKaczmarek** [comment]
correct, I think it's fine to display original input  as well if that's something user may find useful.

**Comment by TravisEz13** [comment]
It is required that you provide adequate test coverage for the code you change. [Contribution Guidelines -Adding test coverage for DSC resources](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#adding-test-coverage-for-dsc-resources)  Please work with us to come up with a plan to cover this code.

**Comment by TravisEz13** [comment]
It is required that you provide adequate test coverage for the code you change. [Contribution Guidelines -Adding test coverage for DSC resources](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#adding-test-coverage-for-dsc-resources)  Please work with us to come up with a plan to cover this code.

**Comment by TravisEz13** [comment]
Please resolve merge conflicts.  For more information see [GitHowTo - 30. Resolving Conflicts](http://githowto.com/resolving_conflicts)

**Comment by TravisEz13** [comment]
If there is no update of the pull request within 7 days, we will close the pull request.  After that, if you have an update, feel free to reopen the pull request.

**Comment by TravisEz13** [comment]
Closing because the pull request has not been updated for feedback given for 21 days.  Once you update the code, please feel free to reopen the pull request.  Thanks, Travis.

**Pull request closed without merging**